Sources/PayPalUI/PaymentButtonColor.swift

```diff
@@ -1,14 +1,109 @@
 import UIKit
 
-enum PaymentButtonColor {
-    case gold, darkBlue
+/// The smart payment button type, this will adjust our images, colors, sizes, etc.
+@objc(PPCPaymentButtonColor)
+public enum PaymentButtonColor: Int, CaseIterable, CustomStringConvertible {
```

Comment:
This can now be:
```
public enum PaymentButtonColor: String, CaseIterable, CustomStringConvertible {

    /// The gold background and blue wordmark, monogram, and black text.
    case gold

    /// The white background and blue wordmark, monogram, and black text.
    case white

    /// The black background and monochrome wordmark, monogram, and white text.
    case black

    /// The silver background and blue wordmark, monogram, and black text.
    case silver

    /// The blue background and white wordmark, blue monogram, and white text.
    case blue

    /// The dark blue background with PayPal Credit wordmark and monogram.
    case darkBlue
```